### DISCUSSION TITLE
Issues starting a new builds

### DISCUSSION THREAD

**Discussion by pravinarr** [codemagic-ci-cd/codemagic-docs#2228] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Tried starting a new build using 
1. api
2. Using UI -> Start new build 

### Expected results

App gets built and pushed to app and play store 

### Actual results

1. App build never gets triggered when tried through api.
2. When tried using the UI, branches get stuck in loading. Even tried to trigger build which were already finished but couldn't.
3. When tried triggering the build using the commit, button never gets enabled.

Mostly similar to the discussions happened here https://github.com/orgs/codemagic-ci-cd/discussions/2092

**Comment by edeuss** [comment]
For me using the UI hitting the "Start new build" button just indicates it has been pressed, then nothing else happens.

**Comment by mmantovani** [comment]
Same problem here.

**Comment by cliffom** [comment]
Experiencing the same issue, an exact mirror of what occurred two weeks ago as outlined here: https://github.com/orgs/codemagic-ci-cd/discussions/2202#discussioncomment-7599073

I can refresh my branches but the `codemagic.yaml` file is not loading in `Select file workflow`.

This is unacceptable as it brings us and many others to a halt. Their issue summary the last time was incorrect as it did impact users authenticated via Oauth. Clearly the root cause has yet to be identified and resolved and we are left in the dark once again.

@himesh-cm @helinanever

**Comment by martynbowis** [comment]
Same issue again for me also

**Comment by ndduy** [comment]
Same issue for me

**Comment by himesh-cm** [comment, marked as answer]
Hello there, everything should be back to normal now.

**Reply by cliffom** [reply]
@himesh-cm can we get a post-mortem on this? Two, identical issues in as many weeks resulting in 12+ hours of downtime for your customers is not acceptable.

**Comment by helinanever** [comment]
Dear users,

Regrettably, as pointed out by some users in this discussion, Codemagic experienced another service disruption similar to the one reported [here](https://github.com/orgs/codemagic-ci-cd/discussions/2202#discussioncomment-7599073).

While the root cause of the issue was different this time, the implications were the same: it was not possible to load the branches of repositories connected via SSH keys or load the codemagic.yaml file to start new builds manually.

The issue lasted approximately from 00:30 UTC to 04:00 UTC on November 29.

Ever since the previous incident, work has been ongoing to improve our infrastructure responsible for the operations mentioned above. There are several updates we’re going to release shortly that will make our infrastructure more robust as well as enhance the overall developer experience for users using codemagic.yaml to configure their app builds.

Additionally, we are working on setting up the [Codemagic status page](https://codemagic.betteruptime.com/) to give our users real-time information about service availability. The current version is just a starting point showing our build machine stack availability, but we are going to add more status checks and system metrics. There will be a public announcement once the status page is fully set up.

Finally, we are sorry for the trouble these incidents have caused and want to reassure you that service reliability is our top priority. Should you have more questions or concerns, please don’t hesitate to get in touch with us via in-app chat (available to users on paid plans).

Best,
Team Codemagic
